gapis/api/vulkan/frame_loop.go

```diff
@@ -105,6 +105,12 @@ type frameLoop struct {
 	descriptorSetLayoutCreated   map[VkDescriptorSetLayout]bool
 	descriptorSetLayoutDestroyed map[VkDescriptorSetLayout]bool
 
+	commandBufferAllocated   map[VkCommandBuffer]bool
+	commandBufferFreed       map[VkCommandBuffer]bool
+	commandBufferChanged     map[VkCommandBuffer]bool
+	commandBufferRecorded    map[VkCommandBuffer]bool // Command buffers recorded during loop.
+	commandBufferNotRecorded map[VkCommandBuffer]bool // Command buffers submitted but not recorded during loop.
```

Comment:
I have changed these to objectToDelete instead of objectCreated in the latest code. This is clearer than the current naming scheme once you have objects getting added to the maps because of object invalidation. Ie, things that need to be destroyed and recreated not because they were created or destroyed inside the loop, but because something else was.

Reply:
yes, I saw your changes, and I can update this once your code is submitted.